internal/karma/ts_web_test.bzl

```diff
@@ -167,6 +173,10 @@ ts_web_test = rule(
             allow_files = True,
             aspects = [sources_aspect],
         ),
+        "setup_files": attr.label_list(
```

Comment:
it feels strange to accept .js files here. it makes it seem like these should user .js files that should be added to the runfiles instead of just filenames that should match the output of a ts_library rule.

Reply:
bootstrap accepts .js files but also adds them to runfiles like so `karma_runfiles += ctx.files.bootstrap` which is what you'd expect when you accept a filename

Reply:
It's definitely an array of entry points, yeah. 

I'm not sure about the label approach though. A label can have many files in it. How do we know which one is the entrypoint then?

Reply:
we error at runtime if you provide a label that is more than one file.
I think this is idiomatic in bazel (currently) - discussed a design with the Bazel TL that does that.